x-pack/plugins/security_solution/public/siem_migrations/rules/components/unknown_migration/index.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import * as i18n from './translations';
+
+const UnknownMigrationComponent = () => {
```

Comment:
Could we export the memo component directly everywhere? It's shorter and it makes it easier to find the references in the IDE.

```
export const UnknownMigration = React.memo(() => {
// [...]
});
UnknownMigration.displayName = 'UnknownMigration';
```

Reply:
Sounds good, will update our components in one of the next PRs.